content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -328,7 +328,7 @@ tokens on behalf of another.
 Kubernetes does not provide an OpenID Connect Identity Provider.
 You can use an existing public OpenID Connect Identity Provider (such as Google, or
 [others](https://connect2id.com/products/nimbus-oauth-openid-connect-sdk/openid-connect-providers)).
-Or, you can run your own Identity Provider, such as CoreOS [dex](https://github.com/coreos/dex),
+Or, you can run your own Identity Provider, such as CoreOS [dex](https://dexidp.io/),
```

Comment:
I don't think it's called CoreOS dex any more
```suggestion
Or, you can run your own Identity Provider, such as [dex](https://dexidp.io/),
```

Reply:
Fair call - fixed